api/services/ocean/spark/schemas/oceanSparkJob.yaml

```diff
@@ -0,0 +1,55 @@
+type: object
+required:
+- id
+- displayName
+- accountId
+- organizationId
+- clusterId
+- controllerClusterId
+description: >
+  The description of an Ocean Spark job
+properties:
+  internalId:
+    $ref: "internalId.yaml"
+  id:
+    $ref: "oceanSparkJobId.yaml"
+  displayName:
+    $ref: "oceanSparkJobDisplayName.yaml"
+  controllerClusterId:
+    $ref: "controllerClusterId.yaml"
+  clusterId:
+    $ref: "oceanSparkClusterId.yaml"
+  accountId:
+    $ref: "accountId.yaml"
+  organizationId:
+    $ref: "organizationId.yaml"
+  createdAt:
+    # I had to write createdAt and updatedAt manually and not use createdAtUpdatedAt.yaml.
+    # Otherwise the object is marked as "recursive". This looks like a redoc bug.
+    $ref: "createdAt.yaml"
+  updatedAt:
+    $ref: "updatedAt.yaml"
+  deletedAt:
+    $ref: "deletedAt.yaml"
+  lastApp:
+    $ref: "oceanSparkApplication.yaml"
+  history:
+    description: "The most recent Spark applications in this job."
+    type: array
+    items:
+      $ref: "oceanSparkApplication.yaml"
+  trends:
+    type: object
+    properties:
+      inputDataBytes:
+        $ref: "oceanSparkJobIntegerTrend.yaml"
+      outputDataBytes:
+        $ref: "oceanSparkJobIntegerTrend.yaml"
+      costCents:
+        $ref: "oceanSparkJobIntegerTrend.yaml"
+      sparkCoresDurationSeconds:
```

Comment:
@HHK1 I replaced the field name `cpuUptimeSeconds` by `sparkCoresDurationSeconds` to match with the application metrics->spark object.
Let me know if this is a bad move and I'll revert this change.

Reply:
No that's what it will be once the API is updated